src/docuquery_ai/ingestion/parser.py

```
@@ -51,108 +57,73 @@
-
-    Returns:
-        A pandas DataFrame containing the CSV data.
-    """
-    # For RAG, we might convert CSV rows to text or handle structured queries separately
     return pd.read_csv(file_path)
 
 
-def parse_excel(file_path: str) -> Dict[str, pd.DataFrame]:
-    """
-    Parses an Excel file into a dictionary of pandas DataFrames, where keys are sheet names.
+def parse_excel(file_path: str) -> Dict[str, "pd.DataFrame"]:
```

Ruff (0.12.2)

Undefined name `pd`

(F821)